packages/namegraph-sdk/src/utils.ts

```diff
@@ -71,29 +71,25 @@ export type NameGraphSuggestion = {
   };
 };
 
-export type NameGraphCategory = {
-  suggestions: NameGraphSuggestion[];
-};
-
 export type NameGraphRelatedCollectionResponse = {
   collection_id: string;
   collection_title: string;
   collection_members_count: number;
 };
 
 export type NameGraphGroupedByCategoryResponse = {
-  categories: NameGraphCategory[];
```

Comment:
Can you please confirm 100% that you validated that these API interface changes are correct?

Reply:
Yes! 💯